libs/extensions/std/lang/src/meta-inf-example-validation.spec.ts

```diff
@@ -0,0 +1,120 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  BlockDefinition,
+  BlockMetaInformation,
+  JayveeServices,
+  createJayveeServices,
+  useExtension,
+} from '@jvalue/jayvee-language-server';
+import {
+  ParseHelperOptions,
+  TestLangExtension,
+  ValidationResult,
+  expectNoParserAndLexerErrors,
+  getTestExtensionBlockForIOType,
+  parseHelper,
+  validationHelper,
+} from '@jvalue/jayvee-language-server/test';
+import { AstNode, AstNodeLocator, LangiumDocument } from 'langium';
+import { NodeFileSystem } from 'langium/node';
+
+import { StdLangExtension } from './extension';
+
+describe('Validation of builtin examples of BlockMetaInformation', () => {
+  let services: JayveeServices;
+  let parse: (
+    input: string,
+    options?: ParseHelperOptions,
+  ) => Promise<LangiumDocument<AstNode>>;
+  let validate: (input: string) => Promise<ValidationResult<AstNode>>;
+
+  let locator: AstNodeLocator;
+
+  const testExtension = new TestLangExtension();
+  const stdExtension = new StdLangExtension();
+
+  async function generateFullJvExample(
+    blockMetaInf: BlockMetaInformation,
+    blockExample: string,
+  ): Promise<string> {
+    // Get name of block in example
```

Comment:
These kind of comments always indicate that the logic could be extracted into a well-named method instead.

Reply:
Yes I could extract it into own method, but honestly this will never be used anywhere else and I personally do not like extracting just for the sake of extracting because it results into a lot of jumping through files when trying to understand the code.  
But if you really prefer me extracting this (and the following parts) into separate methods I will do so.

Reply:
Yes, I think it contributes to better code readability. Keeping methods short and understandable by good method names is good step into the direction of clean code.

You can extract them to the same file, so you'd at least stay in the same file when jumping through the code ;-)